pom.xml

```diff
@@ -13,7 +13,7 @@
     <version>${revision}${changelist}</version>
     <packaging>hpi</packaging>
     <properties>
-        <revision>1.0.1</revision>
+        <revision>4.4.2-1</revision>
```

Comment:
was not sure if incrementals could handle `<revision>${kubernetes-client.version}-1</revision>` which would be cleaner
//cc @jglick

Reply:
Unfortunately no: Maven hard-codes a list of properties which are allowed in this context.